bot/cogs/webhook_remover.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import re
+
+from discord import Colour, Message
+from discord.ext.commands import Cog
+
+from bot.bot import Bot
+from bot.cogs.moderation.modlog import ModLog
+from bot.constants import Channels, Colours, Event, Icons
+
+WEBHOOK_URL_RE = re.compile(r"(discordapp\.com/api/webhooks/)(\d+/)(\S+/?)")
```

Comment:
* Capture the schema if it exists and include the ID in the first group. This avoids the need to use `str.join()` later. 
* Last part doesn't even need to be in a group.
* Ignore case, just in case ;)
```suggestion
WEBHOOK_URL_RE = re.compile(r"((?:https?://)?discordapp\.com/api/webhooks/\d+/)\S+/?", re.I)
```